winlogbeat/docs/getting-started.asciidoc

```diff
@@ -150,6 +150,26 @@ Before starting {beatname_uc}, modify the user credentials in
 +{beatname_lc}.yml+ and specify a user who is
 <<privileges-to-publish-events,authorized to publish events>>.
 
```

Comment:
```suggestion
=== Using a local non-Administrator account to run Winlogbeat

```

It think we should give this section a title and move it out of step 5. I'm open to new location suggestions, but I was thinking to move it to the bottom of this file. And then in step 1 (where we install the service) we link to this section.

Reply:
Added at the end and a link in step1.